client/lib/upgrades/actions/purchases.js

```diff
@@ -150,11 +150,18 @@ function fetchUserPurchases() {
 	} );
 }
 
+function invalidatePurchases() {
+	Dispatcher.handleViewAction( {
+		type: ActionTypes.PURCHASES_INVALIDATE
+	} );
+}
+
```

Comment:
We used the `clearSitePlans()` action and the `REMOVE_SITE_PLANS` action type to do something similar for site plans. Shouldn't we follow the same naming convention here (i.e. `clearPurchases()`and `PURCHASES_REMOVE`)?